src/runtime-rs/crates/runtimes/virt_container/Cargo.toml

```diff
@@ -27,7 +27,7 @@ async-std = "1.12.0"
 
 agent = { path = "../../agent" }
 common = { path = "../common" }
-hypervisor = { path = "../../hypervisor" }
+hypervisor = { path = "../../hypervisor", features = ["cloud-hypervisor"] }
```

Comment:
Could you revert the changes to this file to avoid implicitly fixing #6264 as we're not quite ready for that yet.

Reply:
Updated